cypress/support/constants.js

```diff
@@ -22,3 +22,10 @@ export const THUMBNAIL_EXTENSION = 'image/jpeg';
 export const HOME_PAGE_LOADING_TIME = 3000;
 export const ALL_COLLECTIONS_LOADING_TIME = 3000;
 export const COLLECTION_LOADING_TIME = 3000;
+
+// my list tabs
+export const MY_LIST_TABS = {
```

Comment:
What does this do? If it's for selecting, you should apply "hardcoded" ids to the elements. Most probably you don't need a whole object just for testing.

Reply:
It's the index of tabs. It's not for selecting directly, but to generate the element ids because the element id of the tab contains the index. Otherwise I'll pass in 0, 1, 2 but it's not so clear which represent which.